src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Figures {

```

Comment:
The class 'Figures' seems to be a general concept for figures. If it's intended to be used as a base class for specific figure types, it should be abstract to ensure it cannot be instantiated on its own.